gdk4/src/texture.rs

```diff
@@ -0,0 +1,20 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::Texture;
+use glib::translate::*;
+use glib::IsA;
+
+pub trait TextureExtManual: 'static {
+    #[doc(alias = "gdk_texture_download")]
+    fn download(&self, stride: usize) -> Vec<u8>;
+}
+
+impl<O: IsA<Texture>> TextureExtManual for O {
+    fn download(&self, stride: usize) -> Vec<u8> {
```

Comment:
Not sure if this is nicer than having to pass to a `mut [u8]`?

Reply:
Yes, `&mut [u8]` here and `&[u8]` for upload